test/dockerfiles/Dockerfile.universal

```diff
@@ -5,14 +5,7 @@ ADD $KUMA_ROOT/build/artifacts-linux-amd64/kuma-cp/kuma-cp /usr/bin
 ADD $KUMA_ROOT/build/artifacts-linux-amd64/kuma-dp/kuma-dp /usr/bin
 ADD $KUMA_ROOT/build/artifacts-linux-amd64/kumactl/kumactl /usr/bin
 
-
-RUN mkdir -p /etc/kuma
-ADD $KUMA_ROOT/pkg/config/app/kuma-cp/kuma-cp.defaults.yaml /etc/kuma
-
 RUN mkdir /kuma
-COPY $KUMA_ROOT/tools/releases/templates/LICENSE /kuma
-COPY $KUMA_ROOT/tools/releases/templates/NOTICE /kuma
-COPY $KUMA_ROOT/tools/releases/templates/README /kuma
```

Comment:
Why remove the defaults and the LICENSE/NOTICE/README? I don't think it hurts to have the latter.

Reply:
It's just test docker container, it's not needed